tools/model-compiler/src/main/java/io/spine/tools/compiler/validation/AbstractMethodGroup.java

```diff
@@ -119,4 +119,10 @@ static String validateStatement(String fieldValue, String fieldName) {
                                            FIELD_DESCRIPTOR_NAME, fieldValue, fieldName);
         return codeBlock.toString();
     }
+
+    /** Creates a statement that calls the {@code validateSetOnce} method. */
+    static String validateImmutable() {
```

Comment:
```suggestion
    static String validateSetOnce() {
```

Strictly speaking, this field is not immutable. It is not `MonotonicNonNull` either. It is a partial combination of two.